.github/workflows/ci-review-trigger.yml

```diff
@@ -69,14 +72,7 @@ jobs:
         with:
           app_id: ${{ secrets.GH_APP_DATADOG_VECTOR_CI_APP_ID }}
           private_key: ${{ secrets.GH_APP_DATADOG_VECTOR_CI_APP_PRIVATE_KEY }}
-      - name: Get PR author
-        id: pr
-        uses: tspascoal/get-user-teams-membership@v3
-        with:
-          username: ${{ github.event.issue.user.login }}
-          team: 'Vector'
-          GITHUB_TOKEN: ${{ steps.generate_token.outputs.token }}
-      - name: Get PR comment author
```

Comment:
That is true though it would have required:

- A maintainer to create a fork
- The maintainer to give a non-maintainer write access to their fork
- The maintainer to create a PR from their fork

Typically maintainers create branches in the main repository. Agreed though, it is nice to get rid of.

Reply:
Or steal a GitHub token with write permissions and push changes to a maintainer's branch and merge via comment 😉